### ISSUE TITLE
Relax dependencies or allow `botocore==1.19.52`

### ISSUE THREAD

**Issue by odimko** [heartexlabs/label-studio#1998]
**Is your feature request related to a problem? Please describe.**
I am using the `label-studio` package in combination with [aiobotocore](https://github.com/aio-libs/aiobotocore) and because of the exact pin of `botocore` in `label-studio`, the two cannot be installed together since dependencies cannot be resolved, namely:
`label-studio` requires [botocore==1.19.28](https://github.com/heartexlabs/label-studio/blob/3fc5f1230cf7b10a0cc29c61670b0984fcc0c852/deploy/requirements.txt#L9)
`aiobotocore` requires [botocore>=1.19.52,<1.19.53](https://github.com/aio-libs/aiobotocore/blob/183e268830e68f132f2c6cfe8e8fc9336171e984/setup.py#L10) (for example; there are other versions requiring other `botocore` versions but never the same one as `label-studio`)

**Describe the solution you'd like**
Having exact pins for a package used by other packages makes it really hard to satisfy dependencies, and sometimes, like in my case, it is impossible to do so. Switching to conditional requirements would greatly improve it, e.g. `botocore~=1.19.28`.

**Describe alternatives you've considered**
If the above solution is not possible (why?), could you pin a bit latter version of botocore to `1.19.52`?

**Comment by makseq** [comment]
@odimko Relaxed deps have problems too: pip will try to resolve them automatically and it takes about hour :slightly_frowning_face: Do you know how to avoid this? We had this problem [with google packages](https://github.com/heartexlabs/label-studio/blob/master/deploy/requirements.txt#L11), but strong pins solved it.

**Comment by makseq** [comment]
Also you can try to install [aiobotocore](https://github.com/aio-libs/aiobotocore) after LS installation, it worked for me.

**Comment by odimko** [comment]
Hey @makseq ! thanks for a quick reply :) 
regarding pip being slow, it was a case indeed some time ago, but they've done some big improvements in this regard (e.g. https://github.com/pypa/pip/issues/9187) and dependency resolution sped up. 

For example, I did a `time pip install` of the package and this is the result (using a clean Python virtualenv):
```sh
pip install label-studio==1.3.0 --no-cache-dir  30,61s user 2,99s system 72% cpu 46,392 total
```
for comparison, this is the result when I install all `label-studio`'s reqs with relaxed conditions (also in a new virtualenv):
```sh
pip install -r reqs.txt --no-cache-dir  27,61s user 2,39s 
[reqs.txt](https://github.com/heartexlabs/label-studio/files/8022987/reqs.txt)
system 73% cpu 40,722 total
```
I have used `~=` instead of `==` which is already more flexible for the following packages (the requirements file is attached):
```
boto~=2.49.0
boto3~=1.16.28
botocore~=1.19.28
google-cloud-storage~=1.28.1
google-cloud-logging~=2.6.0
```
So as you can see there is no that big difference 🤔

**Comment by makseq** [comment]
@odimko Could you please check the fix in 1.4.1rc2: `pip install label-studio==1.4.1rc2` ?

**Comment by odimko** [comment]
@makseq , works like a charm! 
```sh
pip install -r reqs.txt --no-cache-dir  44,23s user 3,93s system 54% cpu 1:28,73 total
```
the time has slightly increased because `reqs.txt` contains more than just label-studio as a requirement (I was testing my setup :)).

**Comment by odimko** [comment]
thanks @makseq ! will the fix be released together with the version `1.4.1`?

**Comment by makseq** [comment]
Max Tkachenko commented: It’s already there\!
